src/components/Search.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) => {
+  console.log(props)
```

Comment:
in a "real" code challenge, don't forget to remove console.log and debuggers before you submit